todo.md

```diff
@@ -0,0 +1,11 @@
+# TODOs for json schema generation
```

Comment:
bit_flags should be handled just like regular enums (they are numbers). all `bit_flags` does is assign those numbers differently